src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  background-color: white;
+}
+
+* {
```

Comment:
do not use asterisk for styling - it is too expensive and hard to maintain in bigger apps

Reply:
done, thanks